src/org/dhsdev/flowerknight/FlowerKnight.java

```diff
@@ -75,6 +76,8 @@ public static void mainloop() {
         // While it's open, clear screen and check for events.
         while ( !glfwWindowShouldClose(windowHandle) ) {
 
+            double beforeRender = glfwGetTime();
```

Comment:
Won't reinitializing these variables be rather costly (compared to initializing them before the loop and assigning them here)?